client/post-editor/editor-sharing/publicize-options.jsx

```diff
@@ -86,13 +93,11 @@ const EditorSharingPublicizeOptions = React.createClass( {
 	},
 
 	jetpackModulePopup: function() {
-		let href;
-
-		if ( ! this.props.site || ! this.props.site.jetpack ) {
+		if ( ! this.props.isJetpack ) {
 			return;
 		}
 
-		href = paths.jetpackModules( this.props.site, 'publicize' );
+		const href = paths.jetpackModules( this.props.site, 'publicize' );
```

Comment:
Would be nice to either move away from the `paths` module or at least change it to accept a slug instead of the full `site` object.